src/vue/fields/SelectField.vue

```diff
@@ -103,7 +103,7 @@ export default {
       required: true,
     },
     values: {
-      type: Array,
+      type: [Array, Object],
```

Comment:
`values` prop accept arrays only and should stay like this forever

Reply:
Restored